ext/opentelemetry-ext-asgi/setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
 
 [options.extras_require]
 test =
-    opentelemetry-ext-testutil
+    opentelemetry-test == 0.9.dev0
```

Comment:
Not sure but looked like this needed to be done.

Reply:
yes, there was a refactoring of testutils.